README.md

```diff
@@ -140,8 +140,20 @@ The following inputs can be used as `step.with` keys
 | `stack_destroy` | String | Set to `true` to destroy the stack. Default is `""` - Will delete the elb_logs bucket after the destroy action runs. |
 | `aws_resource_identifier` | String | Set to override the AWS resource identifier for the deployment.  Defaults to `${org}-{repo}-{branch}`.  Use with destroy to destroy specific resources. |
 | `app_directory` | String | Relative path for the directory of the app (i.e. where `Dockerfile` and `docker-compose.yaml` files are located). This is the directory that is copied to the EC2 instance. Default is the root of the repo. |
+| `create_efs` | bool | Toggle to indicate whether to create and EFS and mount it to the ec2 as a part of the provisioning. Note: The EFS will be managed by the stack and will be destroyed along with the stack |
+| `create_ha_efs` | bool | Toggle to indicate whether the EFS resource should be highly available (target mounts in all available zones within region) |
+| `create_efs_replica` | bool | Toggle to indiciate whether a read-only replica should be created for the EFS primary file system |
+| `enable_efs_backup_policy` | bool | Toggle to indiciate whether the EFS should have a backup policy |
+| `efs_zone_mapping` | JSON | Zone Mapping in the form of {\"<availabillity zone>\":{\"subnet_id\":\"subnet-abc123\", \"security_groups\":\[\"sg-abc123\"\]} } |
+| `efs_transition_to_inactive` | string | Indicates how long it takes to transition files to the IA storage class |
+| `replication_configuration_destination` | string | AWS Region to target for replication |
+| `mount_efs_id` | string | ID of existing EFS |
+| `mount_efs_security_group_id` | string | ID of the primary security group used by the existing EFS |
 | `create_keypair_sm_entry` | Boolean | Generates and manage a secret manager entry that contains the public and private keys created for the ec2 instance. |
 | `additional_tags` | JSON | Add additional tags to the terraform [default tags](https://www.hashicorp.com/blog/default-tags-in-the-terraform-aws-provider), any tags put here will be added to all provisioned resources.|
+| `application_mount_target` | string | Directory path in application env to mount directory, default is `data` |
+| `efs_mount_target` | string | Directory path in efs to mount directory to, default is `/` |
```

Comment:
Right but; 
It's not the user needs to specify 12 configurations... 

It's a user HAS 12 configurations they can modify, otherwise the default works just fine

Reply:
@armab 
> Do we need this configurable? I'd assume the EFS target would be / in 90%+ of the cases.

I don't agree. A common use-case I could see would be to have things stored in an NFS that are mounted to specific directories in VMs and containers so that an app could consume it.  One good example would be StackStorm pack files.  Those could be put in a NFS, and then various pods or instances can mount it to a specific location (i.e. where st2 expects the packs to be).

Another example is HeyEmoji. We want NFS to make it more resilient (ephemeral VMs), and HeyEmojy puts its data in the local file system, so if we want to take advantage of an NFS, we need to be able to specify that directory.

Reply:
tldr: i think it should be configurable